spec/valid_email2_spec.rb

```diff
@@ -488,4 +496,45 @@ def set_whitelist
       expect(email.subaddressed?).to be_falsey
     end
   end
+
+  describe "with display_name allowed" do
+    it "is true with email address only" do
+      user = TestUserAllowDisplayName.new(email: "foo@gmail.com")
+      expect(user.valid?).to be_truthy
+    end
+
+    it "is true when address contains a friendly_name" do
+      user = TestUserAllowDisplayName.new(email: "Friendly Name <foo@gmail.com>")
```

Comment:
In case of friendly emails, should it also validate and show errors for cases:

```
Friendly < Name <foo@gmail.com>
Name <fo<o@gmail.com>
Name <fo<o@gmail.c>om>
Name <foo@gmail.c>om>
Name foo@gmail.com
```

Reply:
I've added test cases for those invalid addresses